Description: install data file to meet the need of multi-arch
 Install data files to ${DIR_LIBRARY} instead of ${DIR_SHARE}
Author: Asias He <asias@debian.org>
---
--- opencc-0.4.0.orig/CMakeLists.txt
+++ opencc-0.4.0/CMakeLists.txt
@@ -95,7 +95,7 @@ if (DEFINED SYSCONF_INSTALL_DIR)
 	set (DIR_ETC ${SYSCONF_INSTALL_DIR})
 endif (DEFINED SYSCONF_INSTALL_DIR)
 
-set (DIR_SHARE_OPENCC ${DIR_SHARE}/opencc)
+set (DIR_SHARE_OPENCC ${DIR_LIBRARY}/opencc)
 set (DIR_SHARE_LOCALE ${DIR_SHARE}/locale)
 
 ######## Configuration
